docs/contributor/layered_product_ci.md

```diff
@@ -0,0 +1,191 @@
+# Layered Product Testing with MicroShift
+
+A Layered Product is a software package developed independently from the MicroShift
+source repository and installed as a dependency atop MicroShift. For example, see
+[MicroShift GitOps](https://docs.redhat.com/en/documentation/red_hat_build_of_microshift/4.16/html/installing/microshift-install-rpm#microshift-installing-rpms-for-gitops_microshift-install-rpm).
+
+This document outlines a technique for setting up runtime environments using
+bootable containers to facilitate testing of such software packages. These tests
+can be conducted manually or integrated into the package CI/CD processes. See
+[Image Mode for MicroShift](./image_mode.md) for more information.
+
+> MicroShift includes comprehensive [Automated Functional and Integration Tests](../../test/README.md)
+> for features developed within its source repository.
+
+## Build and Publish MicroShift Container Image
+
+Follow the instructions in the [Build Image](./image_mode.md#build-image) section
+to implement the MicroShift container image layer build procedure.
+
+> Prebuilt MicroShift bootc container images are not currently available for
+> download.
+
+Customize the `Containerfile` file according to the requirements of the layered
+product to be tested. A typical customization would be to select a custom version
+of MicroShift, which may also include pre-released ones that are published at
+[OpenShift Mirror](mirror.openshift.com).
```

Comment:
Good that we have EC/RC build here, too. I guess GA is in the referebce "build-image doc".
Maybe we should provide a small snipped on when to use which version, e.g. use the GA if layered product has an independant release cycle of OpenShift/MicroShift, use EC/RC build when release in lockstep.